packages/flutter_tools/pubspec.yaml

```diff
@@ -23,6 +24,7 @@ dependencies:
   package_config: '>=0.1.5 <2.0.0'
   platform: 1.1.1
   process: 2.0.1
+  quiver: ^0.24.0
```

Comment:
Because we require identical versions across all of our packages, and updating quiver is out of scope for this PR. I can send a follow-up PR upgrading quiver.